src/mate_clj/core.clj

```diff
@@ -156,3 +156,50 @@
   ([pred] (dfilter (complement pred)))
   ([pred coll]
    (dfilter (complement pred) coll)))
+
+(defn dtake-while
+  ([pred]
+     (fn [rf]
```

Comment:
i think all this `fn` block has two extra spaces, it should be just under the `[`
and its relevant to all the functions below

Reply:
done

Reply:
It is still not in the correct indentation check out the other functions in this file.
The block after the arities should be in the same line.

```clojure
(defn dtake-while
  ([pred]
   (fn [rf]
     (fn
       ([] (rf))
       ([result] (rf result))
       ([result input]
        (println pred input "=>" (pred input))
        (if (pred input)
          (rf result input)
          (reduced result))))))
  ([pred coll]
   (lazy-seq
     (when-let [s (seq coll)]
       (println pred (first s) "=>" (pred (first s)))
       (when (pred (first s))
         (cons (first s) (dtake-while pred (rest s))))))))
```